src/navigator/Navigator.tsx

```diff
@@ -750,13 +732,12 @@ function RootStackScreen() {
     <RootStack.Navigator
       screenOptions={{
         backdropComponent: renderBackdrop,
-        handleHeight: animatedHandleHeight,
-        snapPoints: animatedSnapPoints,
-        contentHeight: animatedContentHeight,
+        enableDynamicSizing: true,
+        snapPoints: ['CONTENT_HEIGHT'],
```

Comment:
unfortunately, there is a default snap point of 66%. this means that it is always possible for the bottom sheet to lock into this position. `enableDynamicSizing` adds to the existing snap points instead of overriding them. if we do not set a snap point (or set it to another value), it means that bottom sheet screens will resize to the content, but be able to lock to this arbitrary height too. it looked weird in the wallet connect requests, where you can drag the sheet to lock at 66% : 

https://github.com/valora-inc/wallet/assets/20150449/c7aa76cc-1cf5-4d38-bc36-e5a90eb6686b

to override this behaviour i'm using CONTENT_HEIGHT instead but unfortunately under the hood, this library still uses the deprecated `useBottomSheetDynamicSnapPoints` hook so we still see dev warnings. i think this is the best approach for now, and wait for there to be an update for the lib.

Reply:
Would you mind adding a short comment near this prop for our future selves?

Reply:
yes, added!